### PR TITLE
qtwebkit: (new formula)

### PR DESCRIPTION
**Pull request by s172262**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
-----
`qtwebkit` was cut off from `qt` in https://github.com/Homebrew/homebrew-core/pull/19391.

### CODE REVIEW

**Comment by ilovezfs** [comment]
What is the new upstream repository for this? I'd like to wait for an actual _new_ release from upstream before we commit to distributing a formula for qtwebkit going forward.

**Comment by s172262** [comment]
There is no branch for ver. 5.5.2 in http://code.qt.io/cgit/qt/qtwebkit.git/refs/tags yet.

**Comment by ilovezfs** [comment]
@s172262 it would be good if you could ask upstream what the exact meaning of "They are subject to a separate release" is in https://codereview.qt-project.org/#/c/202006/

What separate release? Where?

**Comment by ilovezfs** [comment]
@s172262 the other thing is I think you need to somehow explicitly tell this to use libpng and libjpeg.

**Comment by s172262** [comment]
`qtwebkit` doesn't need libpng and libjpeg.
```
$ otool -L /usr/local/Cellar/qtwebkit/5.9.1/lib/*.dylib |sed -e "/\/5\.9\.1\//d"|sort|uniq
	/System/Library/Frameworks/AGL.framework/Versions/A/AGL (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/AppKit.framework/Versions/C/AppKit (compatibility version 45.0.0, current version 1561.0.0)
	/System/Library/Frameworks/Carbon.framework/Versions/A/Carbon (compatibility version 2.0.0, current version 158.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1443.13.0)
	/System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics (compatibility version 64.0.0, current version 1125.1.1)
	/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices (compatibility version 1.0.0, current version 822.5.0)
	/System/Library/Frameworks/DiskArbitration.framework/Versions/A/DiskArbitration (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation (compatibility version 300.0.0, current version 1443.14.0)
	/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit (compatibility version 1.0.0, current version 275.0.0)
	/System/Library/Frameworks/IOSurface.framework/Versions/A/IOSurface (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL (compatibility version 1.0.0, current version 1.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.0.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 400.9.0)
	/usr/lib/libicucore.A.dylib (compatibility version 1.0.0, current version 59.1.0)
	/usr/lib/libobjc.A.dylib (compatibility version 1.0.0, current version 228.0.0)
	/usr/lib/libsqlite3.dylib (compatibility version 9.0.0, current version 274.1.0)
	/usr/lib/libxml2.2.dylib (compatibility version 10.0.0, current version 10.9.0)
	/usr/lib/libxslt.1.dylib (compatibility version 3.0.0, current version 3.26.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/local/opt/qt/lib/QtCore.framework/Versions/5/QtCore (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtGui.framework/Versions/5/QtGui (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtMultimedia.framework/Versions/5/QtMultimedia (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtNetwork.framework/Versions/5/QtNetwork (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtPositioning.framework/Versions/5/QtPositioning (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtQml.framework/Versions/5/QtQml (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtQuick.framework/Versions/5/QtQuick (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtSensors.framework/Versions/5/QtSensors (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtSql.framework/Versions/5/QtSql (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtWebChannel.framework/Versions/5/QtWebChannel (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qt/lib/QtWidgets.framework/Versions/5/QtWidgets (compatibility version 5.9.0, current version 5.9.2)
	/usr/local/opt/qtwebkit/lib/libANGLE.1.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/qtwebkit/lib/libJavaScriptCore.1.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/qtwebkit/lib/libWTF.1.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/qtwebkit/lib/libWTRInjectedBundle.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/qtwebkit/lib/libWebCore.1.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/qtwebkit/lib/libWebKit1.1.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/qtwebkit/lib/libWebKit2.1.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/qtwebkit/lib/libleveldb.1.dylib (compatibility version 1.0.0, current version 1.0.0)
```
See `qt.rb`. `qt` includes itself implement like a libpng and libjpeg because `qt` was built with options `-qt-libpng` and `-qt-libjpeg`.

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: We'll need a `make install` or install.sh script.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by s172262** [comment]
"separate release"  is a euphemism. Qt-projects intends to abandons QtWebKit.
QtWebKit had released independently from the start. they revert to type.

**Comment by ilovezfs** [comment]
>QtWebKit had released independently from the start

And where is the upstream for this now? When was the last release? Will there be another release? Where do we report bugs?

**Comment by s172262** [comment]
QtWebKit is a part of WebKit project as in the past. See https://trac.webkit.org/wiki/QtWebKit .

**Comment by ilovezfs** [comment]
That page was last modified five years ago, and for bug reporting points to http://qt-project.org/wiki/ReportingBugsInQt, so it appears there is no active upstream for this project unless I'm missing something.

**Comment by ilovezfs** [comment]
@jakepetroules can you help clarify the situation here?

**Comment by s172262** [comment]
QtWebKit's mailing-list is still alive. 
https://lists.webkit.org/pipermail/webkit-qt/2017-February/004081.html
HEAD repository is http://code.qt.io/cgit/qt/qtwebkit.git/refs/heads .

**Comment by jakepetroules** [comment]
QtWebKit's upstream is at the Qt Project. It's currently deprecated in favor of QtWebEngine, but there are some community members (@annulen) who are working to resurrect it.

**Comment by annulen** [comment]
I'd like to add that intention is to host releases at downloads.qt.io instead of github, however some infrastructure work is yet to be finished before it's possible.

Note that 5.212.0-alpha2 is already packaged by many Linux distros and FreeBSD ports, it is widely used and in general works much better than releases which were shipped with Qt previously.

**Comment by ilovezfs** [comment]
@annulen cool. That's good enough for me. Can we please get some sort of

```
./configure --prefix=#{prefix}
make install
```
or
```
make install PREFIX=#{prefix}
```
or
```
./install.sh #{prefix}
```

or equivalent for this so that we're not storing the logic about what is and isn't supposed to get installed and where it's supposed get installed in the formula itself? That is a requirement for all new formulae.

**Comment by annulen** [comment]
-DCMAKE_INSTALL_PREFIX=..., then make install or ninja install depending on generator.

DESTDIR is supported as well

**Comment by annulen** [comment]
See for example ArchLinux package: https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/qt5-webkit

**Comment by ilovezfs** [comment]
@annulen perfect. Thanks!

@s172262 Please use `std_cmake_args` :)

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: should be `depends_on "qt"` without the `=> [:build, :run]`

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: Can this be passed as an argument to cmake?

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: Needs some kind of assert as well, and should use `begin...ensure`

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: should use `:provided_by_osx`

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: don't think we need this option

File: Formula/qtwebkit.rb
Comment (ilovezfs) [inline Formula/qtwebkit.rb]: goes above pkg-config

**Comment by annulen** [comment]
Please note that https://download.qt.io/official_releases/qt/5.9/5.9.1 contains old QtWebKit

**Comment by ilovezfs** [comment]
@annulen we'll need upstream to remove the `alpha2` from the name of the release in order to package it as the stable spec in Homebrew, if that's what you'd prefer.

**Comment by annulen** [comment]
If you want to package 5.9.1, disregard comments about cmake. Just build it with qmake and use make install, it will use same prefix as Qt, use `INSTALL_ROOT` variable to get `DESTDIR` functionality

**Comment by ilovezfs** [comment]
@annulen it's not that I "want to" package 5.9.1. It's that we have to as a matter of Homebrew policy unless alpha/beta/rc/preview/etc. are not in the version name.

**Comment by annulen** [comment]
I'm not discussing your policies, just trying to help you to do what you want to.

**Comment by ilovezfs** [comment]
@annulen yup. No worries.

@s172262 please follow @annulen's guidance

>Just build it with qmake and use make install, it will use same prefix as Qt, use INSTALL_ROOT variable to get DESTDIR functionality

File: Formula/qtwebkit.rb
Comment (DomT4) [reply Formula/qtwebkit.rb]: Or `should use :provided_by_macos`? Not sure if the plan is to change them at some point or keep both around indefinitely?

Comment (ilovezfs) [reply Formula/qtwebkit.rb]: lol. I didn't even know we had two … :see_no_evil:

Comment (DomT4) [reply Formula/qtwebkit.rb]: Aye. Some of the docs over in the `brew` repo explicitly recommends `keg_only :provided_by_macos` as well 🙈. The language itself has been around since https://github.com/Homebrew/brew/commit/3f9cce0a03e967d2e7bcb7cd16bbc898c1a35708.

Comment (ilovezfs) [reply Formula/qtwebkit.rb]: Maybe they rename back to OS X (iPhone X and all) for 10.14 so I can stop being confused.

File: Formula/qtwebkit.rb
Comment (s172262) [reply Formula/qtwebkit.rb]: `QTDIR` is needed for `build-webkit` script to generate Makefiles and CMakeLists.txt.

File: Formula/qtwebkit.rb
Comment (s172262) [reply Formula/qtwebkit.rb]: Since @annulen said
>Please note that https://download.qt.io/official_releases/qt/5.9/5.9.1 contains old QtWebKit

So I think that everyone should know that System's WebKit which was built from WebKitGTK+ is more secure.

**Comment by s172262** [comment]
Using INSTALL_ROOT makes the following result.
```
$ brew list qtwebkit
/usr/local/Cellar/qtwebkit/5.9.1/usr/local/ (109 files)

$ ls /usr/local/Cellar/qtwebkit/5.9.1/usr/local/Cellar/
qt

$ ls /usr/local/Cellar/qtwebkit/5.9.1/usr/local/Cellar/qt/5.9.2/
lib	libexec	mkspecs	qml	tests

$ ls /usr/local/Cellar/qtwebkit/5.9.1/usr/local/Cellar/qt/5.9.2/lib
QtWebKit.framework		cmake
QtWebKitWidgets.framework	pkgconfig
```

**Comment by ilovezfs** [comment]
@s172262 OK. We'll need some kind of functional `make install` before this can be merged.

**Comment by s172262** [comment]
I'm sorry, I give up this PR.
I know this commit makes errors. I have no idea to avoid these.
```
$ brew audit qtwebkit
qtwebkit:
  * qtwebkit has broken dynamic library links:
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libANGLE.1.dylib
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libleveldb.1.dylib
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib
      /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib
Error: 1 problem in 1 formula
```
The end part of building log: 
```
==> Finishing up
Changing dylib ID of /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/opt/qtwebkit/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libANGLE.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libleveldb.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Changing dylib ID of /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
    to /usr/local/opt/qtwebkit/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libANGLE.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKitWidgets.framework/Versions/5/QtWebKitWidgets
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/libexec/QtWebProcess
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Changing dylib ID of /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
  from libqmlwebkitexperimentalplugin.dylib
    to /usr/local/opt/qtwebkit/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/experimental/libqmlwebkitexperimentalplugin.dylib
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Changing dylib ID of /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
  from libqmlwebkitplugin.dylib
    to /usr/local/opt/qtwebkit/qml/QtWebKit/libqmlwebkitplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/qml/QtWebKit/libqmlwebkitplugin.dylib
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_DesktopBehavior/tst_qmltests_DesktopBehavior
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_DesktopBehavior/tst_qmltests_DesktopBehavior
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_DesktopBehavior/tst_qmltests_DesktopBehavior
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_DesktopBehavior/tst_qmltests_DesktopBehavior
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_DesktopBehavior/tst_qmltests_DesktopBehavior
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_DesktopBehavior/tst_qmltests_DesktopBehavior
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit1.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_WebView/tst_qmltests_WebView
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebKit2.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_WebView/tst_qmltests_WebView
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWebCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_WebView/tst_qmltests_WebView
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libJavaScriptCore.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_WebView/tst_qmltests_WebView
Warning: Could not fix /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/libWTF.1.dylib in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_WebView/tst_qmltests_WebView
Changing install name in /usr/local/Cellar/qtwebkit/5.9.1/tests/tst_qmltests_WebView/tst_qmltests_WebView
  from /private/tmp/qtwebkit-20171015-90027-1rg7hfv/qtwebkit-opensource-src-5.9.1/WebKitBuild/Release/lib/QtWebKit.framework/Versions/5/QtWebKit
    to /usr/local/Cellar/qtwebkit/5.9.1/lib/QtWebKit.framework/Versions/5/QtWebKit
/usr/bin/sandbox-exec -f /tmp/homebrew20171015-4562-ej0iit.sb nice /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/postinstall.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/qtwebkit.rb -s -d --verbose
/usr/local/Homebrew/Library/Homebrew/postinstall.rb (Formulary::FromPathLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/qtwebkit.rb
==> Caveats
We agreed to the WebKit opensource license for you.
If this is unacceptable you should uninstall.

This formula is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

For compilers to find this software you may need to set:
    LDFLAGS:  -L/usr/local/opt/qtwebkit/lib
    CPPFLAGS: -I/usr/local/opt/qtwebkit/include
For pkg-config to find this software you may need to set:
    PKG_CONFIG_PATH: /usr/local/opt/qtwebkit/lib/pkgconfig

==> Summary
🍺  /usr/local/Cellar/qtwebkit/5.9.1: 111 files, 917.7KB, built in 43 minutes 36 seconds
```

**Comment by ilovezfs** [comment]
@s172262 no need to apologize. Thanks for the PR nonetheless :heart: If you do get it working, feel free to reopen this PR or open a new one.

**Comment by sl1pkn07** [comment]
https://github.com/KDE-mac/homebrew-kde/blob/master/qt-webkit.rb

**Pull request closed without merging**
